docs/artifacts.md

```diff
@@ -0,0 +1,175 @@
+<!--
+---
+linkTitle: "Artifacts"
+weight: 201
+---
+-->
+
+# Artifacts
+
+- [Overview](#overview)
+- [Artifact Provenance Data](#passing-artifacts-between-steps)
+  - [Passing Artifacts between Steps](#passing-artifacts-between-steps)
+
+
+
+## Overview
+> :seedling: **`Artifacts` is an [alpha](additional-configs.md#alpha-features) feature.**
+> The `enable-artifacts` feature flag must be set to `"true"` to read or write artifacts in a step.
+
+Artifacts provide a way to track the origin of data produced and consumed within your Tekton Tasks.
+
+## Artifact Provenance Data
+Artifacts fall into two categories:
+
+ - Inputs: Artifacts downloaded and used by a Task.
+ - Outputs: Artifacts created and uploaded by a Task.
+Example Structure:
+```json
+{
+  "inputs":[
+    {
+      "name": "source code", // Category name 
+      "values": [
+        {
+          "uri": "git:jjjsss",
+          "digest": { "sha256": "b35caccc..." }
+        }
+      ]
+    }
+  ],
+  "outputs": [
+    {
+      "name": "build-result", // Category name
+      "values": [
+        {
+          "uri": "pkg:balba",
+          "digest": {
+            "sha256": "df85b9e3...",
+            "sha1": "95588b8f..."
+          }
+        }
+      ]
+    }
+  ]
+}
+```
```

Comment:
Here, you may want to mention things related to `$(steps.artifacts.<artifact-name>.path)`. i.e. How does a user artifacts in their Steps.

Reply:
I guess you have that information in the example below. I would suggest breaking it an showing things as needed.